openpype/hosts/houdini/plugins/publish/validate_primitive_hierarchy_paths.py

```diff
@@ -104,3 +113,60 @@ def get_invalid(cls, instance):
                 "(%s of %s prims)" % (path_attr, len(invalid_prims), num_prims)
             )
             return [output_node.path()]
+
+    @classmethod
+    def repair(cls, instance):
+        """Add a default path attribute Action.
+
+        it's a helper action more than a repair action,
+        which used to add a default single value.
+        """
+
+        output_node = instance.data.get("output_node")
```

Comment:
Be aware that output node could be **not** set which will still fail in repair down below. Should we capture that case where no path is set (or not valid node path is set) and just log an error and return?

Reply:
what about this ? 
It will make us avoid a few things

```python
sop_path = rop_node.parm('sop_path').eval()
if not sop_path:
    return 
 output_node = hou.node(sop_path)
```

Reply:
> what about this ? It will make us avoid a few things
> 
> ```python
> sop_path = rop_node.parm('sop_path').eval()
> if not sop_path:
>     return 
>  output_node = hou.node(sop_path)
> ```

maybe it's better to have log.message to tell user it will not perform repair action if there is no sop_path ?

Reply:
You'll likely want this: `rop_node.parm("sop_path").evalAsNode()`
Right?